### PR TITLE
NestJS Advanced

### PR DESCRIPTION
**Pull request by Skysplit**

### CODE REVIEW

**Comment by Skysplit** [comment]
@qbart would you mind reviewing this, please?

File: content/web_development/skills/Framework and language/nestjs/advanced.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+## 📦 Websockets
+
+### 🎓 Learn
+
+- 📗 [Using multiple nodes](https://socket.io/docs/v4/using-multiple-nodes)
+- 📗 [Websockets](https://docs.nestjs.com/websockets/gateways)
+
+### 🎤 Interview
+
+- What is sticky session? When do you need to apply it?
```
Comment (qbart) [inline content/web_development/skills/Framework and language/nestjs/advanced.md:101]: do you think we should ask more about websockets?

File: content/web_development/skills/Framework and language/nestjs/advanced.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+### 📝 Katas
+
+- Create a custom validator that checks against uniqueness of user email in database. Use dependency injection provided by NestJS.
+
+## 📦 Websockets
+
+### 🎓 Learn
+
+- 📗 [Using multiple nodes](https://socket.io/docs/v4/using-multiple-nodes)
+- 📗 [Websockets](https://docs.nestjs.com/websockets/gateways)
```
Comment (qbart) [inline content/web_development/skills/Framework and language/nestjs/advanced.md:97]:
```suggestion
- 📗 [Websockets](https://docs.nestjs.com/websockets/gateways)
- 📗 [A Beginner's Guide to WebSockets](https://www.youtube.com/watch?v=8ARodQ4Wlf4)
```

**Review by qbart — APPROVED** [review]
LGTM, I was wondering if maybe we should add more info/questions about websockets

**Comment by Skysplit** [comment]
@qbart I'll try to add some more content related to websockets then. Will come back to you tomorrow!

File: content/web_development/skills/Framework and language/nestjs/advanced.md (hunk 1 above)
Comment (Skysplit) [reply content/web_development/skills/Framework and language/nestjs/advanced.md:107]: added some more questions about WS and SSE as well

**Review by qbart — APPROVED** [review]
:shipit:

**Pull request merged**